frontend/src/component/project/Project/PaginatedProjectFeatureToggles/TableEmptyState/TableEmptyState.tsx

```diff
@@ -3,12 +3,15 @@ import { Box } from '@mui/material';
 import { TablePlaceholder } from 'component/common/Table';
 
 interface ITablePlaceholderProps {
-    total?: number;
+    show: boolean;
     query?: string;
 }
 
-export const Placeholder: FC<ITablePlaceholderProps> = ({ total, query }) => {
-    if (total !== 0) {
+export const TableEmptyState: FC<ITablePlaceholderProps> = ({
```

Comment:
can the parent decide whether to show this component or not instead of the component looking at a boolean param?